Assets/MixedRealityToolkit.SDK/Features/Utilities/PointerUtils.cs

```diff
@@ -144,6 +146,128 @@ public static IEnumerable<IMixedRealityPointer> GetPointers()
+        public static void SetGazePointerBehavior(PointerBehavior pointerBehavior)
+        {
+            if (CoreServices.InputSystem.FocusProvider is IPointerPreferences pointerPreferences)
+            {
+                pointerPreferences.GazePointerBehavior = pointerBehavior;
+                foreach (InputSourceType sourceType in Enum.GetValues(typeof(InputSourceType)))
+                {
+                    pointerPreferences.SetPointerBehavior<GGVPointer>(Handedness.Any, sourceType, pointerBehavior);
+                }
+            }
+            else
+            {
+                WarnAboutSettingCustomPointerBehaviors();
+            }
+        }
+
+        /// <summary>
+        /// Sets the pointer behavior for pointer of type T, for all input types.
+        /// </summary>
+        /// <typeparam name="T">All pointer types that equal or derive from this type will be set.</typeparam>
+        /// <param name="pointerBehavior">Desired <seealso cref="Microsoft.MixedReality.Toolkit.Input.PointerBehavior"/>.</param>
+        /// <param name="handedness">Specify handedness to restrict to only right, left.</param>
+        public static void SetPointerBehavior<T>(PointerBehavior pointerBehavior, Handedness handedness = Handedness.Any) where T : class, IMixedRealityPointer
+        {
+            foreach (InputSourceType type in Enum.GetValues(typeof(InputSourceType)))
+            {
+                SetPointerBehavior<T>(pointerBehavior, type, handedness);
+            }
+        }
+
+        /// <summary>
+        /// Sets the behavior for the given pointer type and input type.
+        /// </summary>
+        /// <typeparam name="T">All pointer types that equal or derive from this type will be set.</typeparam>
+        /// <param name="pointerBehavior">Desired <seealso cref="Microsoft.MixedReality.Toolkit.Input.PointerBehavior"/>.</param>
+        /// <param name="sourceType">Allows setting different behaviors for different input types (hands, controllers, etc.)</param>
+        /// <param name="handedness">Specify handedness to restrict to only right, left.</param>
+        public static void SetPointerBehavior<T>(PointerBehavior pointerBehavior, InputSourceType sourceType, Handedness handedness = Handedness.Any) where T : class, IMixedRealityPointer
+        {
+            if (CoreServices.InputSystem.FocusProvider is IPointerPreferences preferences)
+            {
+                preferences.SetPointerBehavior<T>(handedness, sourceType, pointerBehavior);
+            }
+            else
+            {
+                WarnAboutSettingCustomPointerBehaviors();
+            }
+        }
+
+        private static void WarnAboutSettingCustomPointerBehaviors()
+        {
+            Debug.LogWarning("Setting custom pointer behaviors only works if the input system is using the default MRTK focus provider. " +
+                "Are you using a custom Focus Provider?");
```

Comment:
>Focus Provider [](start = 40, length = 14)

Nit: "Are you using a custom Focus Provider that doesn't implement IPointerPreferences"

To give folks who see the error a next step on what to do (i.e. implement IPointerPreferences)